fs/fs.go

```diff
@@ -13,14 +13,15 @@ import (
 	"text/template"
 	"time"
 
-	"github.com/jvatic/goja-babel"
+	babel "github.com/jvatic/goja-babel"
```

Comment:
Why rename the import?

Reply:
My editor (vscode) did this automatically! Will revert. Not sure why it does that by default